main.tex

```diff
@@ -415,12 +416,19 @@ \section{Development history and status}
 JPLHorizons module has been implemented as part of the \texttt{sbpy}
 project\footnote{\url{http://sbpy.org}} with support from NASA PDART grant
 80NSSC18K0987. Further Solar System-related services are planned to be added to
-\astroquery through this support.
-
-\astroquery has received support from the Google Summer of Code
+\astroquery through this support. \astroquery has also received support from the Google Summer of Code
 program, with two students (co-authors Madhura Parikh and Simon Liedtke)
 from 2013--2014.
 
+Due to its nature as a open developed ``big tent'' package, \astroquery intentionally
```

Comment:
I don't like "big tent"; I had never heard the colloquialism and had to look it up
```suggestion
Due to its nature as an openly developed package, \astroquery intentionally
```

Reply:
agreed